chef/cookbooks/dhcp/recipes/default.rb

```diff
@@ -63,18 +78,27 @@
 intfs = [Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").interface]
 address = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").address
 
-d_opts = node[:dhcp][:options]
+require "ipaddr"
+admin_addr = IPAddr.new(address)
+
+if admin_addr.ipv4?
```

Comment:
See my previous comment about maybe referencing an ip_version variable. I also wonder if this precludes us from doing dual-stack, meaning we would have to come back to this and refactor it again.

Reply:
Yeah, I have a feeling there might be a bunch or refactoring required for true dual stack. Currently crowbar only expects 1 network of each type. So I think a bunch of changes maybe be required for it to have more then 1 network for each type. I think once we're gone through single stack, we'll have a very good idea of how much of crowbar we need to touch.

Unfortunately if I was more experienced with Crowbar, maybe I'd be able to tell you outright. But this is a new journey into the crux of crowbar for me too ;)